index.js

```diff
@@ -503,6 +510,7 @@ module.exports = {
           pull.drain(
             (msg) => {
               const groupId = msg.value.content.recps[0]
+              // TODO: also check if it's in one of the epoch tips
```

Comment:
hmm there might be edge cases with this like if the new epoch is more than 1 epoch in the future and replication is slow, and how exactly getTipEpochs acts in that case. but that's definitely a bug we can fix in the future, and the worst that happens is the user gets removed from the group on the clientside